about.html

```diff
@@ -0,0 +1,377 @@
+				<img
+					class="indicator"
+					src="./public/images/indicator.svg"
+					alt="Line Break"
+				/>
+
+				<div>
+					<p class="text-metal-grey">
+						The logo of World Conference on Arabic Language 2022 was
+						decided through the logo competition from 14th December
+						to 22nd December.
+						<span class="hidden md:block">
+							After online public voting and Selection Committee's
+							judging the logo from Hana eld won the prize. <br />
+							After consultation with Mr. Mohamed Ali we optimized
+							the prize-winning logo for this year's conference as
+							following
+						</span>
+					</p>
+				</div>
+
+				<div
+					class="about-logo-image w-full max-w-600px border border-coral-grey"
+				>
+					<img
+						src="./public/images/logo-arabic2.jpg"
+						alt="logo"
+						class="about-logo-image w-full px-8 py-4 md:py-8 md:px-16"
+					/>
+				</div>
+			</div>
+		</section>
+
+		<!-- Past Events section -->
+		<section class="text-center past-event flex" id="pastEvent">
+			<div>
+				<h2>See the past Arabic Language Conferences</h2>
+				<img
+					class="indicator"
+					src="./public/images/indicator02.svg"
+					alt="Line Break"
+					style="text-align: center"
+				/>
+			</div>
+			<p>
+				Take a look at the last two Arabic Language Conferences which
+				took place in Dubai and Cairo Cities.
+			</p>
+			<ul>
+				<li class="event01">
+					<h3>2018</h3>
+					<h4>Arabic Language Conference in Dubai</h4>
+				</li>
+				<li class="event02">
+					<h3>2020</h3>
+					<h4>Arabic Language Conference in Cairo</h4>
+				</li>
+			</ul>
+		</section>
+
+		<!-- partner section  -->
+		<section
+			id="partner"
+			class="relative flex flex-col items-center bg-heavy-metal py-8 px-4 mobile-show"
+		>
+			<h2
+				class="partner-title relative text-center text-2xl font-medium text-metal-grey after:absolute after:left-[calc(50%-48px/2)] after:-bottom-3 after:h-[2px] after:w-12 after:bg-coral-red"
+			>
+				Partner
+				<img
+					class="indicator"
+					src="./public/images/indicator.svg"
+					alt="Line Break"
+				/>
+			</h2>
+			<div
+				class="partners-items-container mt-5 flex w-75 flex-wrap items-center justify-center gap-4"
+			>
+				<div class="partners-img-container max-w-120px">
+					<img
+						src="./public/images/partner-mozilla.png"
+						alt="mozilla"
+					/>
+				</div>
+				<div class="partners-img-container max-w-120px">
+					<img
+						src="./public/images/partner-google.png"
+						alt="google"
+					/>
+				</div>
+				<div class="partners-img-container max-w-120px">
+					<img src="./public/images/partner-naver.png" alt="naver" />
+				</div>
+				<div class="partners-img-container max-w-120px">
+					<img
+						src="./public/images/partner-daumkakao.png"
+						alt="daumkakao"
+					/>
+				</div>
+				<div class="partners-img-container max-w-120px">
+					<img
+						src="./public/images/partner-airbnb.png"
+						alt="airbnb"
+					/>
+				</div>
+			</div>
```

Comment:
- [ ] I highly recommend using the `ul` tag while displaying any list on the webpage and make sure each item on the list is wrapped into the `li` tag. This will help to follow the best HTML practices. :wink: